### ISSUE TITLE
SEO - Blog.

### ISSUE THREAD

**Issue by AdanPorto** [aHub-Tech/hub-blog#8]
Olá. Podemos falar mais dobre SEO para o blog?

Podemos colocar categorias, Imagens, descrição, comentarios, Backlinks, Posts relacionados. ETC...

Como podemos falar sobre isso?

**Comment by RodrigoDornelles** [comment]
> Olá. Podemos falar mais dobre SEO para o blog?
> 
> Podemos colocar categorias, Imagens, descrição, comentarios, Backlinks, Posts relacionados. ETC...
> 
> Como podemos falar sobre isso?

Isso é um bom ponto. algumas coisas já tenho em mente de como implementar para melhorar, ou até mesmo já esta parcialmente pronto porém não utilizado.

Seria interessante novas postagens, para fazer sentido poder começar a categorizar e relacionar postagens.

Por ser puramente frontend o blog não possui um banco de dados para os comentários, mas podemos utilizar de algum serviço para isso e incorporar ao site.

### O Que já existe ###

- [ ] Categorias
- [X] Imagens
- [X] Descrição
- [ ] Comentários
- [ ] Backlinks
- [ ] Posts relacionados
- [X] Site map
- [X] Feed

**Comment by AdanPorto** [comment]
Temos que instalar as ferramentas do Google. GTM e Search Console.

**Comment by caio-emidio** [comment]
Alguma novidade sobre isso meninos?
"Nao deixa a issue morrer, nao deixa a issue acabar" (Alcione Codes)

**Comment by RodrigoDornelles** [comment]
> Alguma novidade sobre isso meninos? "Nao deixa a issue morrer, nao deixa a issue acabar" (Alcione Codes)

Estou esperando fechar mais posts para trazer melhorias na paginação e relacionar posts, categorias.. etc

**Comment by AdanPorto** [comment]
Podemos criar um post manual de estrutura para como criar um post para ajudar no SEO.

**Comment by RodrigoDornelles** [comment]
> Podemos criar um post manual de estrutura para como criar um post para ajudar no SEO.

Sim, isso pode até ser uma postagem para o blog, agregando mais conteúdo

**Comment by RodrigoDornelles** [comment]
@AdanPorto eu considero ferramentas do google muito absivas quanto a privacidade, o que acha em vez de usar o google console para fazer analises de SEO, utilizarmos o do cloudflare, por ser menos invasivo?

**Comment by AdanPorto** [comment]
O search console é a ferramenta que ajuda da indexação do conteúdo e das paginas... O que colhe os dados é o analytics.
